CONTRIBUTING.md

```diff
@@ -1 +1,58 @@
-Check [waspc README](waspc) for detailed contributing guidelines.
+# Contributing to Wasp
+
+Wasp's compiler is built with Haskell. As an output, you'll get a web application written in React and NodeJS. So there are three ways for you to contribute: 
+- [Haskell-related](#haskell-related)
+- [Web dev](#web-dev) (React, Node, HTML/CSS, database and so on)
+- [Tutorials or Example apps](#tutorials-or-example-apps)
+- [Documentation](#documentation)
+
+## Before you begin 
+
+- Check the [**Getting Started**](https://wasp-lang.dev/docs) guide to get familiar with Wasp's fundamentals. And ideally, build an app from the [**Pick a Tutorial**](https://wasp-lang.dev/docs/pick-a-tutorial) page
+- Figure out what you'd like to help with. It can be code, documentation, tutorials, etc. More on this is below.
+- Join our Discord [![**Discord**](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) for faster communication and feedback. Also, we'll be glad to help you find the issue you'll enjoy working on, depending on your skill set!
+
+Let's jump right in!
+
+---
+
+## Haskell-related
+
+Wasp compiler is implemented in Haskell, but you will also see a lot of Javascript and other web technologies because the Wasp compiler transpilers Wasp code into them.
```

Comment:
Yeah, not sure about this one. Grammarly thinks otherwise. 

![image](https://user-images.githubusercontent.com/10496082/192810078-2c8515b6-4e82-40bc-842a-bc982e88e5af.png)